lib/rubocop/cop/rspec/empty_line_after_subject.rb

```diff
@@ -16,18 +16,15 @@ module RSpec
       #   let(:foo) { bar }
       class EmptyLineAfterSubject < Cop
         extend AutoCorrector
-        include RuboCop::RSpec::BlankLineSeparation
+        include RuboCop::RSpec::EmptyLineSeparation
 
-        MSG = 'Add empty line after `subject`.'
+        MSG = 'Add empty line after `%<subject>s`.'
```

Comment:
The rest of the messages begin with "Add **an** empty line".

Reply:
Fixed.